cmd/signing.go

```diff
@@ -31,31 +32,29 @@ func init() {
 	SigningCommand = cli.Command{
 		Name:  "signing",
 		Usage: "Provides several tools useful for out-of-band signing",
+		Before: func(c *cli.Context) error {
```

Comment:
We can also use `Before` here and below to check the `Args` length and make sure it matches what we expect, returning an `error` if it does not.

Reply:
Yep, but I feel it's a little bit clearer if we validate the arguments in the subcommand right in the place where we use them.